packages/crypto/yarn.lock

```diff
@@ -289,9 +289,15 @@ acorn-walk@^8.1.1:
   integrity sha512-k+iyHEuPgSw6SbuDpGQM+06HQUa04DZ3o+F6CSzXMvvI5KMvnaEqXe+YVe555R9nn6GPt404fos4wcgpw12SDA==
 
 acorn@^8.4.1, acorn@^8.8.0:
+<<<<<<< HEAD
```

Comment:
There are various conflicts in the pushed lockfiles now...

Reply:
yeah I noticed too, I thought I had regenerated them and apparently it didn't actually do it, so I went through and manually did it afterward.